logstash-core/build.gradle

```diff
@@ -27,27 +27,12 @@ buildscript {
     }
 }
 
-plugins {
-    id "jacoco"
-    id "org.sonarqube" version "4.3.0.3225"
-}
 
-apply plugin: 'jacoco'
```

Comment:
I couldnt figure out a way to avoid downloading these deps at test time. I'll have to circle back to this.